zipkin-web/src/main/resources/templates/v2/index.mustache

```diff
@@ -22,7 +22,7 @@
       <label for="timeStamp">End time</label>
       <input class="form-control input-sm date-input" type="text">
       <input class="form-control input-sm time-input" type="text">
-      <input class="" id="endTs" name="endTs" type="hidden" value="{{endTs}}">
+      <input class="timestamp-value" id="endTs" name="endTs" type="hidden" value="{{endTs}}">
```

Comment:
I see, you want this function to apply to all timestamps, not just this one `endTs`.